docs/cli/enterprise.md

```diff
@@ -0,0 +1,219 @@
+
+- **Location**:
+  - **Linux**: `/etc/gemini-cli/settings.json`
+  - **Windows**: `C:\ProgramData\gemini-cli\settings.json`
+  - **macOS**: `/Library/Application Support/GeminiCli/settings.json`
+  - The path can be overridden using the `GEMINI_CLI_SYSTEM_SETTINGS_PATH` environment variable.
+- **Control**: This file should be managed by system administrators and protected with appropriate file permissions to prevent unauthorized modification by users.
+
+By using the system settings file, you can enforce the security and configuration patterns described below.
+
+## Restricting Tool Access
+
+You can significantly enhance security by controlling which tools the Gemini model can use. This is achieved through the `coreTools` and `excludeTools` settings.
+
+### Allowlisting with `coreTools`
+
+The most secure approach is to explicitly add the tools and commands that users are permitted to execute to an allowlist. This prevents the use of any tool not on the approved list.
+
+**Example:** Allow only safe, read-only file operations and listing files.
+
+```json
+{
+  "coreTools": ["ReadFileTool", "GlobTool", "ShellTool(ls)"]
+}
+```
+
+### Blocklisting with `excludeTools`
+
+Alternatively, you can add specific tools that are considered dangerous in your environment to a blocklist.
+
+**Example:** Prevent the use of the shell tool for removing files.
+
+```json
+{
+  "excludeTools": ["ShellTool(rm -rf)"]
+}
+```
+
+**Security Note:** Blocklisting with `excludeTools` is less secure than allowlisting with `coreTools`, as it relies on blocking known-bad commands, and clever users may find ways to bypass simple string-based blocks. **Allowlisting is the recommended approach.**
+
+## Managing Custom Tools (MCP Servers)
+
+If your organization uses custom tools via [Model-Context Protocol (MCP) servers](../core/tools-api.md), it is crucial to understand how server configurations are managed to apply security policies effectively.
+
+### How MCP Server Configurations are Merged
+
+Gemini CLI loads `settings.json` files from three levels: System, Workspace, and User. When it comes to the `mcpServers` object, these configurations are **merged**:
+
+1.  **Merging:** The lists of servers from all three levels are combined into a single list.
+2.  **Precedence:** If a server with the **same name** is defined at multiple levels (e.g., a server named `corp-api` exists in both system and user settings), the definition from the highest-precedence level is used. The order of precedence is: **System > Workspace > User**.
+
+This means a user **cannot** override the definition of a server that is already defined in the system-level settings. However, they **can** add new servers with unique names.
+
+### Enforcing a Catalog of Tools
+
+The security of your MCP tool ecosystem depends on a combination of defining the canonical servers and adding their names to an allowlist.
+
+#### More Secure Pattern: Define and Add to Allowlist in System Settings
```

Comment:
I think we should also call out `includeTools` for MCP servers, really helps to be secure when using 3p MCP servers, like only allow read tools instead of mutating tools